app/templates/components/chart-toggle.html

```diff
@@ -0,0 +1,18 @@
+<p class="text-corporate-blue text-center text-h2">
```

Comment:
You can consider using [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) and `<figcaption>` here to semantically connect the chart with this title!

Reply:
Jep. Good point